docker-compose.yml

```diff
@@ -15,6 +15,18 @@ services:
       - "3001:3001"
     env_file:
       - .env.local
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfile.server
+      args:
+        - USE_CRON=true
+    depends_on:
+      - db
+  rake:
```

Comment:
Recapping our zoom! Plan B: if we can't get the initial invocation of the `rake check:all` command in `docker-entrypoint.`🤐  or into the cronjob itself, then let's plop an inline comment here explaining that this `rake` container really only serves to execute the first command for developer experience, all subsequent calls are via the cronjob